hathor/p2p/states/ready.py

```diff
@@ -88,24 +78,23 @@ def on_enter(self) -> None:
         self.lc_ping.start(1, now=False)
         self.send_get_peers()
 
-        for plugin in self.plugins.values():
-            plugin.start()
+        self.sync_manager.start()
 
     def on_exit(self) -> None:
         if self.lc_ping.running:
             self.lc_ping.stop()
 
-        for plugin in self.plugins.values():
-            plugin.stop()
+        if self.sync_manager.is_started:
+            self.sync_manager.stop()
 
-    def get_sync_plugin(self) -> NodeSyncTimestamp:
-        return cast(NodeSyncTimestamp, self.plugins[self.SYNC_PLUGIN_NAME])
+    def prepare_to_disconnect(self) -> None:
+        self.sync_manager.stop()
```

Comment:
Is it safe enough to assume `sync_manager` will be started here? Or should we add the same `if` as in the `on_exit` method? `if self.sync_manager.is_started:`